http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -466,8 +464,12 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
   # For older jupyter images, jupyter_delocalize.py is using 127.0.0.1 as welder's url, which won't work now that we're no longer using `network_mode: host` for GCE VMs
   docker exec $JUPYTER_SERVER_NAME sed -i  "s/127.0.0.1/welder/g" /etc/jupyter/custom/jupyter_delocalize.py
 
+  # make sure home directory is owned by jupyter-user
+  docker exec -u root ${JUPYTER_SERVER_NAME} chown -R jupyter-user:users $JUPYTER_HOME
+
   log 'Starting Jupyter Notebook...'
-  retry 3 docker exec -d ${JUPYTER_SERVER_NAME} ${JUPYTER_SCRIPTS}/run-jupyter.sh ${NOTEBOOKS_DIR}
+  sleep 5
```

Comment:
I'm not sure why this sleep is needed, and it makes me nervous. Without it, the `run-jupyter.sh` script was not actually taking effect, making me think there's some race condition. Will keep investigating this, but wanted to get a PR out

Reply:
Think I figured this out, removed the sleep